gridt/src/app/movements/movements-detail/movements-detail.page.html

```diff
@@ -13,9 +13,9 @@
       <ion-row>
         <ion-col size-md="6" offset-md="3">
           <h2>{{ (movement$ | async)?.name }}</h2>
-          <p><strong>{{ (movement$ | async)?.short_description }}</strong></p>
-          <p *ngIf="(movement$ | async )?.description">{{ (movement$ | async)?.description }}</p>
-
+          <p class="shortdescription">{{ (movement$ | async)?.short_description }}</p>
```

Comment:
Why did you remove the strong?

Reply:
The font type is now included in the css class for the shortdescription as well. It comes out the same.

Reply:
That is a code design decision. I don't know how much you know about this, but if you are unfamiliar with semantic html, I would recommend reading the following article. I don't necessarily argue one way or the other, but I do think that it is useful to make it in an informed manner.

https://medium.com/adalab/the-importance-of-semantic-html-78e74fb75ff0

Reply:
Interesting consideration and good to become aware of. However, I think that in this case, the <strong> tag has a layout function than rather than a semantic one. Therefore, I would still leave it in the scss file.

Reply:
Okay, then I'll approve your pull request.